packages/frontend/amp/components/elements/AdComponent.test.tsx

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import { render } from 'react-testing-library';
+import { AdComponent } from './AdComponent';
+
+describe('AdComponent', () => {
+    const edition = 'UK';
+    const section = '';
+    const contentType = '';
+    const commercialConfig = {
+        useKrux: true,
+        usePrebid: true,
+    };
+    const commercialProperties = {
+        UK: { adTargeting: [] },
+        US: { adTargeting: [] },
+        AU: { adTargeting: [] },
+        INT: { adTargeting: [] },
+    };
+    const kruxURL =
+        'https://cdn.krxd.net/userdata/v2/amp/2196ddf0-947c-45ec-9b0d-0a82fb280cb8?segments_key=x&kuid_key=kuid';
```

Comment:
Should these URLs be hard-coded here? Probably fine, I just call it out to make sure we considered whether this is the right place for these config strings :-)